docroot/modules/custom/mass_views/src/NodeArgumentAccessHandler.php

```diff
@@ -43,7 +43,7 @@ public function __construct(AccountInterface $account, RouteMatchInterface $curr
    */
   public function access(AccountInterface $account) {
     $node = $this->currentRouteMatch->getParameter('node');
-    if ($node->access('view', $account)) {
+    if (!empty($node) && $node->access('view', $account)) {
```

Comment:
This was a fix for an error I was seeing on the collections page. Related to a recent fix here: #1801